tests/unit/api/bucketDelete.js

```diff
@@ -14,7 +14,7 @@ const canonicalID = 'accessKey1';
 const authInfo = makeAuthInfo(canonicalID);
 const namespace = 'default';
 const bucketName = 'bucketname';
-const postBody = new Buffer('I am a body');
+const postBody = Buffer.from('I am a body', 'binary');
```

Comment:
No, it's actually utf8 only. I did the same mistake when I did the binary buffers effort in metadata and bucketclient.